src/RMP/Translate/lang/programmes/en.po

```diff
@@ -1044,6 +1044,10 @@ msgstr "Last Played"
 msgid "latest"
 msgstr "Latest"
 
+#. English: "Latest episode".
+msgid "latest_episode"
+msgstr "Latest Episode"
```

Comment:
I was going by designs. Ran it by a designer and they said yes it should be lower case.
Sorry.